src/components/MessagesList/MessagesList.vue

```diff
@@ -267,6 +267,9 @@ export default {
 			this.$store.dispatch('easeMessageList', { token: oldToken })
 			this.stopFetchingOldMessages = false
 			this.messagesGroupedByDateByAuthor = this.prepareMessagesGroups(this.messagesList)
+			this.$nextTick(() => {
+				this.isScrollable = this.$refs.scroller?.scrollHeight > this.$refs.scroller?.clientHeight
```

Comment:
Also nextTick should be in `messagesList` watcher, because it depends on its render.
Maybe move this check to `scrollToBottom()` method? It already has nextTick and check for scroller to exist

```js
data() {
	isScrollable: false,
}